theano/tensor/nnet/abstract_conv.py

```diff
@@ -1613,7 +1612,7 @@ def perform(self, node, inp, out_):
         if isinstance(mode, tuple):
             pad = tuple(int(mode[i]) for i in range(self.convdim))
             mode = "valid"
-            new_img = numpy.zeros((img.shape[0], img.shape[1]) +
+            new_img = np.zeros((img.shape[0], img.shape[1]) +
                                   tuple(img.shape[i + 2] + 2 * pad[i]
```

Comment:
I have some problems in contributing to theano, i am not good in git. could you help me out